src/invidious/channels/channels.cr

```diff
@@ -151,23 +152,18 @@ def get_batch_channels(channels, refresh = false, pull_all_videos = true, max_th
   return final
 end
 
-def get_channel(id, refresh = true, pull_all_videos = true)
-  if channel = Invidious::Database::Channels.select(id)
-    if refresh && Time.utc - channel.updated > 10.minutes
-      channel = fetch_channel(id, pull_all_videos: pull_all_videos)
-      Invidious::Database::Channels.insert(channel, update_on_conflict: true)
-    end
```

Comment:
I think it could be a good idea to keep a time-based condition for channel refresh. Currently channel infos are stored once in the DB and never updated, which could make user wonder why some channels have outdated infos (e.g description). Something like twice a week, maybe?

Reply:
I didn't realize this when making this change, but we already have a job for this https://github.com/iv-org/invidious/blob/master/src/invidious/jobs/refresh_channels_job.cr

Looking through the code now, it looks like it refreshes every channel's data every 30 minutes. Could probably be tweaked 😬

Reply:
Note that the channel refreshing job can be disabled by config, making this function the only way to update the channel's data.